package.json

```diff
@@ -1,16 +1,21 @@
 {
-  "name" : "hexy",
-  "version" : "0.2.11",
-  "description" : "hexdump, binary pretty-printing",
-  "author" : "Tim Becker <tim.becker@kuriositaet.de>",
+  "name": "hexy",
+  "version": "0.2.11",
+  "description": "hexdump, binary pretty-printing",
+  "author": "Tim Becker <tim.becker@kuriositaet.de>",
   "license": "MIT",
-  "main" : "./hexy.js",
+  "main": "./hexy.js",
   "repository": {
     "type": "git",
     "url": "git://github.com/a2800276/hexy.js.git"
   },
-  "scripts" : {
-    "test" : "node ./test.js"
+  "scripts": {
+    "test": "node ./test.js"
   },
-  "bin": {"hexy" : "./bin/hexy_cmd.js"}
+  "bin": {
+    "hexy": "./bin/hexy_cmd.js"
+  },
+  "dependencies": {
```

Comment:
I'll change this to devDependencies ... hoping that won't wreak havoc

Reply:
Just installed 0.3.0 and the types and dependencies are working fine.  Thanks!

Reply:
now I can delete my branch.